.travis.yml

```diff
@@ -0,0 +1,58 @@
+sudo: required
```

Comment:
This is identical to `meetup-web-platform .travis.yml` except doesn't run the coveralls step yet